lib/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Util = require('util');
+
+const Config = require('../config');
+const MoviesServer = require('./server');
+
+MoviesServer.start(() => {
```

Comment:
Usually when we name things, we don't append "what" it is after the variable name. So instead of `MoviesServer`, it's just `Movies`. Same with `MoviesModel`, `AWSLibrary`, etc.